javascript/index.js

```diff
@@ -14,52 +15,122 @@ let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
 function printTime() {
-  // ... your code goes here
+  getMinutes(); //return minutes
+  getSeconds()// return seconds
+  twoDigitsNumber(minutes)
+  twoDigitsNumber(seconds)
+
+  minDec.innerHTML = minutes[0]
+  minUni.innerHTML = minutes[1]
+  secDec.innerHTML = seconds[0]
+  secUni.innerHTML = seconds[1]
+
 }
 
 function printMinutes() {
-  // ... your code goes here
+  getMinutes(); //return minutes
+  twoDigitsNumber(minutes)
+
+  minDec.innerHTML = minutes[0]
+  minUni.innerHTML = minutes[1]
 }
 
 function printSeconds() {
-  // ... your code goes here
+  getSeconds()// return seconds
+  twoDigitsNumber(seconds)
+
+  secDec.innerHTML = seconds[0]
+  secUni.innerHTML = seconds[1]
 }
 
 // ==> BONUS
 function printMilliseconds() {
   // ... your code goes here
+  // Didn't manage to do it :(
 }
 
 function printSplit() {
-  // ... your code goes here
+  
+  let splitBtn = document.getElementsByClassName("split");
+  
+  splitBtn.addEventListener("click", function(){
+    event.preventDefault();
+    let recordedTime = splitClick() // return 12:34
+    let newLi = document.createElement("li");
+    newLi.innerHTML = recordedTime;
+    splits.appendChild(newLi);
+  })
 }
 
 function clearSplits() {
-  // ... your code goes here
+  let resetBtn = document.getElementsByClassName("reset");
+
+  resetBtn.addEventListener("click", function(){
+    event.preventDefault();
+    minDec.innerHTML = "0"
+    minUni.innerHTML = "0"
+    secDec.innerHTML = "0"
+    secUni.innerHTML = "0"
+
+    while (splits.firstChild) {
+      splits.removeChild(splits.firstChild)
+    }
+  }
 }
+  
 
 function setStopBtn() {
-  // ... your code goes here
+  btnLeft.addEventListener("click", function (){
```

Comment:
You already have an addEventListener at the bottom of the code, you want to run these functions that modify the DOM there. You want to have addEventListeners that call functions and not functions that call addEventListeners. Otherwise the functions are never being called.